cart-service/src/main/java/com/petclinic/cartsservice/businesslayer/CartService.java

```diff
@@ -8,4 +8,6 @@
 public interface CartService {
 
     public Mono<CartResponseModel> getCartByCartId(String cartId);
+    Mono<Void> clearCart(String cartId);
```

Comment:
I think it should return a Flux since inside a cart there is a flux of ProductResponseModels

Reply:
just updated it